packages/cli/src/commands/server/eventsSocket.ts

```diff
@@ -0,0 +1,161 @@
+/**
+ * (c) Facebook, Inc. and its affiliates. Confidential and proprietary.
```

Comment:
Is this header appropriate here? We're not adding them to any new files and this project is MIT based. Not a lawyer, but "confidential" is a bit concerning :D

Reply:
I think I copied that just from some of the other CLI files, will double check😅

Reply:
Just dropped the header, like in most files. Some files do have still this header though (e.g. messageSocket)